core/src/main/java/io/kestra/core/repositories/TriggerRepositoryInterface.java

```diff
@@ -17,5 +19,11 @@ public interface TriggerRepositoryInterface {
     Trigger save(Trigger trigger);
 
     void delete(Trigger trigger);
+
+    ArrayListTotal<Trigger> find(Pageable from, String query, String namespace);
+
+    default Function<String, String> sortMapping() throws IllegalArgumentException {
+        return s -> s;
+    }
```

Comment:
I copied it from user / role / namespace repository, it is used for JDBC sorts to work since we're incompatible in term of case. The UI send a camelCase sort which must get translated to lower_case for jdbc to match actual columns (while in ES it is in camelCase so no change needed)

Reply:
Agree with Function.identity, I just wanted to keep the same codebase as for other repo but seems okay